lib/plugins/nwc/src/lib.rs

```diff
@@ -225,9 +259,52 @@ impl SdkNwcService {
             }
         };
 
+        if client.connection.receive_only && !matches!(req.params, RequestParams::MakeInvoice(_)) {
```

Comment:
Do we block also other commands such as ListTransactions on receive only?
Should we also return error to the caller here?